.github/workflows/end-to-end.yml

```diff
@@ -0,0 +1,47 @@
+name: New End-to-End Test
+run-name: fedbiomed-e2e-tests
+permissions: write-all
+on:
+  schedule:
+    - cron: '30 1 * * *'
+  workflow_dispatch:
+  push:
+    branches:
+      - 'develop'
+      - 'master'
+      - 'feature/end-to-end-test-machinery'
+      - 'feature/ubuntu-e2e-fail-debug'
+
+jobs:
+  e2e-test-auto:
+    name: e2e tests
+    strategy:
+      fail-fast: false # Importance ?
```

Comment:
I don't know either :) but I know that `fail-fast = false` important to continue on test in other slave in case of failure. However, the default value is `false`. So I will remove it.